les/client_handler.go

```diff
@@ -45,6 +46,47 @@ type clientHandler struct {
 	closeCh  chan struct{}
 	wg       sync.WaitGroup // WaitGroup used to track all connected peers.
 	syncDone func()         // Test hooks when syncing is done.
+
+	//@rayyuan
+	gatewayFeeCache *gatewayFeeCache
+}
+
+type gatewayFeeEtherbase struct {
```

Comment:
More information will likely be added to this type, especially because the gateway fee will need to be 
split into different parts for each supported fee currency in the future. As such a more generic name would be better. Something like `GatewayFeeInformation`